near.js

```diff
@@ -233,7 +233,7 @@ class Near {
         options.changeMethods.forEach((methodName) => {
             contract[methodName] = async function (args) {
                 args = args || {};
-                const response = await near.scheduleFunctionCall(0, options.sender, contractAccountId, methodName, args);
+                const response = await near.scheduleFunctionCall(1000000000, options.sender, contractAccountId, methodName, args);
```

Comment:
what does this number mean? could you pull this out into const with a comment?

Reply:
Added constant and comment.